app/routes/search_organisation.js

```diff
@@ -1,4 +1,7 @@
-import AuthorizeRoute from './authorize';
-
-export default AuthorizeRoute.extend({});
+import AuthorizeRoute from "./authorize";
 
+export default AuthorizeRoute.extend({
+  setupController() {
+    this.controllerFor("application").set("showSidebar", false);
+  }
```

Comment:
do we need a reset controller for this ? @abulsayyad123 . I think we do.

Reply:
either of them works.

Reply:
No, i'm saying we need to revert this property to whatever it was, while controller is exiting. 
Because its application level property. It might affect other places.

Reply:
I am sorry, I miss understood. Yes, you are right. I intentionally did not reset it because from this page we redirect to `account_details` page in all possible cases. And in that page it is already doing this reset.

Reply:
okkay, but ideally we should do it. What if someone navigates through the url to lets say home ?
sidebar won't be visible. That'll cause issue over there.